[ZT] WARP client logs - add new FAQ (#4059)
cloudflare/cloudflare-docs: content/cloudflare-one/faq/cloudflare-tunnels-faq.md

Tunnel credentials file '/root/.cloudflared/928655cc-7f95-43f2-8539-2aba6cf3592d.json' doesn't exist or is not a file
```

## How do I contact support?

> ### I am having an issue with a locally-managed tunnel

Reviewer: ```suggestion
### I am having an issue with a locally-managed tunnel.
```

- Make sure that the `cloudflared daemon` is updated to the [latest version](https://github.com/cloudflare/cloudflared).

- Gather any relevant error/access logs from your server.

> ### I am having issues accessing resources through WARP

Reviewer: ```suggestion
### I am having issues accessing resources on private networks through the WARP client.
```
as a separate note, do you think it makes sense to phrase this in the first person? I'm thinking those who will have issues in this case will likely be end users, not admins themselves

> While [connecting private networks](/cloudflare-one/connections/connect-networks/private-net/) with Cloudflare Tunnels, end users will connect to your resources using the [WARP client](/cloudflare-one/connections/connect-devices/warp/).

Reviewer: ```suggestion
If your end users are experiencing issues while accessing resources on private networks [exposed via Cloudflare Tunnel](/cloudflare-one/connections/connect-networks/private-net/), ensure you include WARP client logs when contacting Cloudflare support. These logs will help Cloudflare support understand the overall architecture of your machine and networks.
```

In order to help Cloudflare support understand the overall architecture of your machine and networks, send the WARP Client logs with your request.

> #### macOS

Reviewer: ```suggestion
#### Retrieving WARP client logs on macOS
```

Reviewer: actually — should we have a separate section under the WARP docs outlining how to retrieve logs and then link out to that section from here? looks like information that users may find helpful regardless of tunnel issues

Author: I was debating whether to put this in the WARP FAQ or Tunnel FAQ. I don't know if the logs are helpful for anything else besides Support issues.

Reviewer: yeah, might be worth moving the logs info to the WARP FAQ and linking out to them. Surely users will look for logs-related info for WARP issues outside of tunnel scenarios
